tools/sidekick/config.json

```diff
@@ -101,6 +101,17 @@
       "url": "https://milo.adobe.com/tools/tag-selector",
       "isPalette": true,
       "paletteRect": "top: 150px; left: 7%; height: 675px; width: 85vw;"
+    },
+    {
+      "containerId": "tools",
+      "id": "version-history",
+      "title": "Version History",
+      "environments": [ "edit" ],
+      "url": "https://milo.adobe.com/tools/version-history",
+      "isPalette": true,
+      "passReferrer": true,
+      "paletteRect": "top: auto; bottom: 20px; left: 20px; height: 498px; width: 460px;",
+      "includePaths": [ "**.docx**" ]
     }
```

Comment:
I believe we added support for xlsx files too in version history. Can you please add .xlsx also in includePaths?